en/docs/reference/pricing.md

```diff
@@ -2,229 +2,145 @@
-- The web application then makes **1 API call to the [Flight Status API](#flightStatusAPI)** in Choreo. This is counted as **1 step**.
-- The Flight Status API in Choreo then makes **1 call to the [Real-Time Flight API](#realtimeFlightAPI)** with the search criteria entered by the client. This is counted as **1 step**. 
-- The Real-Time Flight API then invokes the database to retrieve the information.
+This section summarizes the features supported for each plan:
 
-- **Assuming the entire execution time for the process to obtain the flight status takes less than 500ms** (starting from the web application making a call to Choreo, and Choreo making a call to the backend API),this process thereby results in a **total of 2 steps**.
+#### Developer and operational
 
-<br/>
+Choreo supports the following developer and operational features for all the plans:
 
-### Example 3: Webhook
-An airport has its flight information display system (FIDS) in Choreo.  This system involves APIs and webhooks.
+- Managing the source code via a GitHub repository
 
-#### Scenario
-This scenario describes a hypothetical maintenance process carried out by airline maintenance. The airline uses Choreo to generate alerts by using webhooks when the aircraft maintenance status is updated. 
-The Aircraft Maintenance team carries out maintenance for each flight before take-off. A supervisor updates the maintenance status of the aircraft on a [Aircraft Maintenance Sheet](#AircraftMaintenanceSheet), which is a Google sheet. Upon updating the sheet, the [Aircraft Status Notifier](#aircraftNotifWebhook) webhook on Choreo gets triggered and updates the [Flight Scheduling Database](#flightSchedulingDatabase) that confirms if the aircraft is ready for take off,  whether it will be delayed due to maintenance, or whether it is not suitable for take off.
+- Continuous integrations (CI)
 
-#### Glossary
+- Continuous delivery (CD)
 
-- <a class="pricing" id="aircraftNotifWebhook">Aircraft Status Notifier</a>:A webhook in Choreo that listens to any events that trigger when the Aircraft maintenance sheet is updated.
-- <a class="pricing" id="AircraftMaintenanceSheet">Aircraft Maintenance Sheet</a>: A Google sheet that maintains the list of aircraft, their information, the maintenance schedule, and the maintenance status.
-- <a class="pricing" id="flightSchedulingDatabase">Flight Scheduling Database</a>:The database that includes details about  flights, the aircraft assigned to the flight route, and their maintenance state.
+- Deployments to cloud platforms are supported for the three plans as follows:
 
-<br/>
+   | **Free** | **Pay As You Go** | **Enterprise** |
+   |----------|-------------------|----------------|
+   | Azure    | Azure<br/>AWS<br/>GCP<br/>VMWare<br/>OpenShift<br/>Tanzu<br/>Any Kubernetes cluster |Azure<br/>AWS<br/>GCP<br/>VMWare<br/>OpenShift<br/>Tanzu<br/>Any Kubernetes cluster |
```

Comment:
The table in the preview mode looks weird. Shall we have like bullet points inside the table?

Reply:
Though it appears that way in the Git preview, it looks as follows in the local build and will be rendered this way.

<img width="740" alt="Screen Shot 2023-01-05 at 12 07 32" src="https://user-images.githubusercontent.com/26239861/210717229-50969b8d-ee80-45a9-b20d-727362ce4f5e.png">